Documentation/README.md

```diff
@@ -15,22 +15,42 @@ For more information on how to get the config app id from launch check this [doc
 
 ## Configuration
 To enable advertising identifier features in the sample app, follow these steps:
-1. Set your `ENVIRONMENT_FILE_ID` in MainApp.java.
-2. Update the value for key `gms_ads_app_id` located in the `secrets.xml` at [aepsdk-sample-app-android/Sample-App/app/src/main/res/values](../Sample-App/app/src/main/res/values/secrets.xml) with a valid Google AdMob app ID.
+1. Update the value for key `gms_ads_app_id` located in the `secrets.xml` at [aepsdk-sample-app-android/Sample-App/app/src/main/res/values](../Sample-App/app/src/main/res/values/secrets.xml) with a valid Google AdMob app ID.
     - See Google's [quick start reference](https://developers.google.com/admob/android/quick-start) on how to get your AdMob app ID. See step 3 of the [Configure your app](https://developers.google.com/admob/android/quick-start#import_the_mobile_ads_sdk) section for a free public test app ID from Google.
     - Any real key values in the `secrets.xml` file should **not** be committed to the repository.
-3. Uncomment the implemention code using [find and replace](https://www.jetbrains.com/help/idea/finding-and-replacing-text-in-project.html#replace_search_string_in_project) to replace all instances of:
+2. Because the ad ID features are commented out to prevent misconfigured app crashes, you need to uncomment the implemention code using [find and replace all](https://www.jetbrains.com/help/idea/finding-and-replacing-text-in-project.html#replace_search_string_in_project) to replace all instances of:
```

Comment:
By default, the ad ID features are commented out in the sample app. You need to uncomment the implementation code by using [find and replace all]...

Something like that?  No need to mention crash  here.

Reply:
That makes sense, thank you! Wording has been updated